src/popup/pages/Popups/AccountList.vue

```diff
@@ -0,0 +1,175 @@
+        @click="confirm()"
+      />
+    </template>
+  </Modal>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType } from '@vue/composition-api';
+import type {
+  IAppData,
+  IPermission,
+} from '../../../types';
+import {
+  PERMISSION_DEFAULTS,
+  POPUP_CONNECT_ADDRESS_PERMISSION,
+  POPUP_CONNECT_TRANSACTIONS_PERMISSION,
+} from '../../utils';
+import { useState } from '../../../composables/vuex';
+
+import Modal from '../../components/Modal.vue';
+import BtnMain from '../../components/buttons/BtnMain.vue';
+import TransactionInfo from '../../components/TransactionInfo.vue';
+import CheckMark from '../../../icons/check-mark.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    Modal,
+    BtnMain,
+    TransactionInfo,
+    CheckMark,
+  },
+  props: {
+    app: { type: Object as PropType<IAppData>, required: true },
+    access: {
+      type: Array,
+      default: () => ([
+        POPUP_CONNECT_ADDRESS_PERMISSION,
+      ]),
+    },
+    resolve: { type: Function as PropType<() => void>, required: true },
+    // eslint-disable-next-line no-unused-vars
+    reject: { type: Function as PropType<(e: Error) => void>, required: true },
+  },
+  setup(props, { root }) {
+    const permission = useState<IPermission>('permissions', props.app.host);
+    const appName = computed(() => permission.value?.name || props.app.name);
+    const recipient = {
+      wallet: 'Superhero Wallet',
+    };
+
+    function confirm() {
+      root.$store.commit('permissions/addPermission', {
+        ...PERMISSION_DEFAULTS,
+        ...props.app,
+        ...permission.value,
+      });
+      props.resolve();
+    }
+
+    function cancel() {
+      props.reject(new Error('Rejected by user'));
+    }
+
+    return {
+      POPUP_CONNECT_ADDRESS_PERMISSION,
+      POPUP_CONNECT_TRANSACTIONS_PERMISSION,
+      appName,
+      confirm,
+      cancel,
+      recipient,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
+@use '../../../styles/typography';
+
+.connect {
+  .transaction-info {
+    margin-bottom: 16px;
+  }
+
+  .subtitle {
+    @extend %face-sans-15-medium;
+
+    margin-top: 24px;
+    margin-bottom: 16px;
+    color: variables.$color-grey-light;
+    text-align: center;
+
+    .app-name {
+      color: variables.$color-white;
+    }
+  }
+
+  .permissions {
+    margin: 16px 0;
+
+    .title {
+      @extend %face-sans-15-medium;
+
+      display: flex;
+      align-items: center;
+      padding-bottom: 4px;
+      color: variables.$color-grey-dark;
+
+      .icon {
+        width: 24px;
+        height: 24px;
+        color: variables.$color-success;
+        padding-right: 4px;
+      }
+    }
+
+    .description {
+      @extend %face-sans-15-regular;
+
+      display: block;
+      padding-bottom: 6px;
+      color: variables.$color-white;
+      text-align: left;
+    }
+
+    .message {
+      color: variables.$color-warning;
+    }
```

Comment:
This can be removed and instead you can use `color-warning` CSS class (see `global.scss` file).

Reply:
This is not working for the popups. Popups don't include the global styles in my opinion

Reply:
`global.scss` is merged with all other files. Why wouldn't it be not merged?

Reply:
We had a discussion regarding this, and were able to fix this. Thanks @pstachel